components/eamxx/src/physics/ml_correction/eamxx_ml_correction_process_interface.cpp

```diff
@@ -187,23 +214,16 @@ void MLCorrection::run_impl(const double dt) {
         // Apply all the adjustment to a single phase based on surface temperature
         Kokkos::single(Kokkos::PerTeam(team), [&] {
           auto T_icol = ekat::subview(T_mid,icol);
-          if (T_icol(m_num_levs-1)>273.15) {
+          if (T_icol(num_levs-1)>273.15) {
```

Comment:
Is `num_levs` interchangeable with `m_num_levs`?

Reply:
We need to save the local variable `num_levs=m_num_levs` for running on Device.